# Effects of KN95 mask on microbiome and oxygen saturation

*Djakim Latumalea<sup>a</sup>, Azadeh Pirzadeh<sup>a</sup>, Kai Lin<sup>a</sup>, Peter Riesebos<sup>a</sup>, Hossain Shahadat<sup>a</sup>*

*<sup>a</sup>Institute for Life Science & Technology, Hanze University of Applied Sciences, Zernikeplein 11, 9747 AS Groningen, The Netherlands*

**Abstract here**

*KKN95 | FFP2 | SpO<sub>2</sub> | Microbiome | Facemask*


## Introduction
Since the new severe acute respiratory syndrome coronavirus (SARS-CoV-2), that causes coronavirus disease 2019 (CODID-19), measures were taken to limit the spreading of the virus (1). One of the taken measures, is the use of face masks. Surgical masks reduce the emission of virus particles in the environment (*). Therefore, masks have become recommended or mandatory in the Western-Europe. Due to surgical mask use, several skin changes and other negative effects have been observed (2). Negative effects include discomfort, a changing microbiome balance and a decrease in oxygen saturation, which can cause headaches and breathing problems (3) and other symptoms that are related to cognitive decline (15). A surgical mask can influence the blood oxygen saturation levels when resting (15) and when exercising (18). The blood oxygen saturation levels can decrease, because CO2-rich exhaled air is trapped in the mask and does not disperse, resulting in an environment richer in CO2, and poorer in O2 (16, 17). Several studies report a change in microbiome (19, 20, 21), although the composition can differ. Wearing a mask for a prolonged time can lead to an increase in skin temperature, redness and a marginal increase in the secretion of sebum content (!), which can influence the pH-value of the skin, which in turn, influences the skin barrier. This may also be the reason, why some people observe the development of acne (24).

In this study we compared the effects of certified surgical masks on the skin microbiome and the oxygen saturation with a baseline measurement. We took samples of the check to analyze the microbiome composition. In particular, we looked the dominance of Propionibacterium acnes (P. acnes). In addition, we studied the relation between skin microbiome changes and oxygen saturation. 

## Materials and methods
Five volunteers (4 male and 1 female) participated in the study. All volunteers were between 24 and 36 years old, available for 4 weeks and did not have prior skin disease. They did not use antibiotics or had an infection in the last month. They were also not immune-compromised or suffered from other medical conditions that would make wearing a facemask impractical. However, one subject used hormonal supplements (Levothyroxine), but used them the whole experiment. Volunteers were given a KN95/FFP2 mask from KUBAOYA and they wore them for 8 hours 5 days a week. The volunteers were requested to clean their hands with alcohol-based sanitizer for 20 to 30 seconds or with water and soap for 40 to 60 seconds. They were also given instructions on how to wear the mask, to take off the mask, and how to store the mask. The masks were replaced every day with new masks. The volunteers were also asked to fill in a diary every day. This way, we analyzed factors that could influence the experiment.

To examine the influence of wearing facemasks on blood oxygen saturation, we measured and collected data by using a CMS50D1 pulse oximiter from CONTEC. To make accurate measurements, volunteers followed the protocol that was provided by CONTEC. The volunteers measured both the left and right index finger to detect discrepancies in the values. They were instructed to record the value of the right index finger.

Subjects did not bathe/shower 12-24 hours before sampling, and same sampler for all the experimental weeks. We used sterile gloves each sample collection to minimize sample cross-contamination and soak a sterile cotton swab in 1.5 ML PBST Buffer in microcentrifuge tube. Rub the soaked swab on the pre-measured skin area ( 4cm X 4cm) with zigzag strokes with constant pressure. Cut the swab head to 1-3 cm in the length to the remaining microcentrifuge tube. Vortex microcentrifuge tube for 1 minute and centrifuge for 17,000g 15minutes in room temperature. Remove 1 ml supernatant and transfer the sample to a new microcentrifuge tube without swab head.
We resuspended the sample in microcentrifuge and transferred 50ul to LB medium for liquid culture, 50ul to BHI medium for liquid culture, 50ul to LB plate, 50ul to BHI plate, 50ul to blood agar and put them into the 37°C incubator for 3 days and storge in the fridge to store it.

Put the remaining liquid in the microcentrifuge to liquid nitrogen. We followed the protocol to isolate the DNA from third edition December 2014 QIAamp® DNA Micro Handbook. We used the tissues protocol. To get higher concentration of DNA, we added 1 ul of carrier RNA at step 5 and applied 20 ul distilled water at step 12. Using Qubit® dsDNA BR Assay Kits and nano drop detected the quantities of the sample.
 
To detect the microbiome, we used Oxford Nanopore 16S Barcoding Kit (SQK-RAB204) for 16S sequencing and followed the protocol version RAB_9053_v1_revN_14Aug2019.


The data were analyzed using Python. Pandas was the most utilized module and the data was visualized using Holoviz/Panel to create a dashboard.

## Results

## Discussion

## Acknowledgments

## References

